src/content/documentation/publish/distribute-manifest-versions.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: sidebar
+title: Distribute Manifest V2 and V3 extensions
+permalink: /documentation/publish/distribute-manifest-versions/
+topic: Publish
+tags: [add-ons, beginner, tutorial, webextensions]
+contributors: [rebloor]
+last_updated_by: rebloor
+date: 2022-11-17
+---
+
+<!-- Page Hero Banner -->
+
+{% capture page_hero_banner_content %}
+
+# Distribute Manifest V2 and V3 extensions
```

Comment:
The title is different than the google doc.  Not commenting if this is better or worse, just that isn't a difference between the workshopped draft doc and final webpage.

Reply:
I modified to fits better with other titles on EW